### PR TITLE
Update application config to be the new format - Closes #6569

### LINKED ISSUE #6569: Update application config to be the new format
### Description

Update application config to be more module centric as below interface.
```
applicationConfigSchema = {
  ...
  genesis: {
    communityIdentifier: ‘’,
    modules: [moduleName]: Record<string, unknown>,
  },
  generator: {
    password: "",
    modules: [moduleName]: Record<string, unknown>
  },
}
```

### Acceptance Criteria
- Configs schema and fixtures are updated
- All unit tests are fixed

### Additional Information
Related issues #6554

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?

This PR resolves #6569

### How was it solved?

- Configs schema and fixtures are updated
- All unit tests are fixed

### How was it tested?

- Run all unit tests

### CODE REVIEW

**Review by shuse2 — APPROVED** [review]

File: framework/src/schema/application_config_schema.ts [hunk 1]
```diff
@@ -12,10 +12,94 @@
...
+	},
+	additionalProperties: { type: 'object' },
+};
+
+const rewardsConfigSchema = {
+	type: 'object',
+	required: ['milestones', 'offset', 'distance'],
+	description: 'Object representing LSK rewards milestone',
+	properties: {
+		milestones: {
```
Comment (ManuGowda) [inline framework/src/schema/application_config_schema.ts:29]: the naming has changed, maybe reflect the change? https://github.com/LiskHQ/lips-staging/pull/94/files#diff-dfeda675218ab1dec2be752051c6b2a96bd201a0d704ebbb438c31b32312fb25R63
@IkerAlus its `brackets` now?

File: framework/src/schema/application_config_schema.ts [hunk 2]
```diff
@@ -12,10 +12,94 @@
...
+		properties: {
+			encryptedPassphrase: {
+				type: 'string',
+				format: 'encryptedPassphrase',
+			},
+			address: {
+				type: 'string',
+				format: 'hex',
+			},
+			hashOnion: {
```
Comment (ManuGowda) [inline framework/src/schema/application_config_schema.ts:64]: doesn't this go to random module specific config?

File: framework/src/schema/application_config_schema.ts [hunk 3]
```diff
@@ -357,6 +382,89 @@ export const applicationConfigSchema = {
 				},
 			},
 		},
+		genesis: {
```
Comment (ManuGowda) [inline framework/src/schema/application_config_schema.ts:385]: some of the props should go into specific module?

File: framework/src/schema/application_config_schema.ts (hunk 2 above)
Comment (nazarhussain) [reply framework/src/schema/application_config_schema.ts:64]: Yes I believe module specific config must be splitted. But I am not sure which one are those and when to do it.   @shuse2 may explain here.

File: framework/src/schema/application_config_schema.ts (hunk 3 above)
Comment (nazarhussain) [reply framework/src/schema/application_config_schema.ts:385]: Yes these should go, but may be @shuse2  plan is to do it with the issues for each module when we start.

File: framework/src/schema/application_config_schema.ts (hunk 1 above)
Comment (nazarhussain) [reply framework/src/schema/application_config_schema.ts:29]: When the implementation of that LIP starts, we should make that change.

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**